src/java/picard/illumina/MarkIlluminaAdapters.java

```diff
@@ -64,15 +64,27 @@
  * @author Tim Fennell (adapted by mborkan@broadinstitute.org)
  */
 @CommandLineProgramProperties(
-        usage = "Reads a SAM or BAM file and rewrites it with new adapter-trimming tags.\n" +
-                "Clear any existing adapter-trimming tags (XT:i:).\n" +
-                "Only works for unaligned files in query-name order.\n"+
-                "Note: This is a utility program and will not be run in the pipeline.\n",
-        usageShort = "Reads a SAM or BAM file and rewrites it with new adapter-trimming tags",
+
+        usage = MarkIlluminaAdapters.USAGE_SUMMARY + MarkIlluminaAdapters.USAGE_DETAILS,
+        usageShort = MarkIlluminaAdapters.USAGE_SUMMARY,
         programGroup = Illumina.class
 )
 public class MarkIlluminaAdapters extends CommandLineProgram {
 
+    static final String USAGE_SUMMARY = "Reads a SAM or BAM file and rewrites it with new adapter-trimming tags.  ";
+    static final String USAGE_DETAILS = "<p>Reads a SAM or BAM file and rewrites it with new adapter-trimming tags.  " +
+            "Initially will clear any existing adapter-trimming tags (XT:i:) in the optional tag region of a SAM file.  " +
+            "The SAM/BAM file must be sorted by query name.  You can use Picard's " +
```

Comment:
I don't think we need to point people to how to sort a SAM file. If they're playing with adapter trimming they know what they're doing, at least that far.